03_simon_says/simon_says.rb

```diff
@@ -1 +1,34 @@
-#write your code here
+def echo(echo)
+  echo
+end
+
+def shout(hello)
+  hello.upcase
+end
+
+
+def repeat(hello, repeat = 2)
+  (hello + " ") * (repeat - 1) + hello
```

Comment:
Similarly, this is confusing.

```suggestion
def repeat(hello, times = 2)
  (hello + " ") * (times - 1) + hello
```